app/controllers/org_chart_controller.rb

```diff
@@ -0,0 +1,69 @@
+class OrgChartController < ApplicationController
+  before_action -> { user_has_position("Secretary") }, only: [:create, :destroy]
+  before_action :authenticate_user!
+
+  def index
+    @charts = []
+    division_names = User.uniq.pluck(:division).reject(&:nil?)
+    division_names.each do |name|
+      data_table = GoogleVisualr::DataTable.new
+      data_table.new_column('string', 'Name'   )
```

Comment:
Unnecessary spacing detected.<br>Space inside parentheses detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.